### PR TITLE
Restore branch name in `clean-conversation-headers`

### LINKED ISSUE #3808: Base branch missing from PR view
Version: 20.12.9

Perhaps the is a feature, but I was pretty sure the merge destination used to show up. Now it doesn't. Or maybe doesn't when it's the main branch of the parent repo? 

Anyway, it confused a few of us this morning so thought it'd be good to open a bug.

https://github.com/Safecast/safecastapi/pull/743

With extension on:

Screen_Shot_2020-12-10_at_0_15_34

With extension off:

Screen_Shot_2020-12-10_at_0_15_42

### PR DESCRIPTION
**Pull request by yakov116**
Closes #3808 

## Test URLs
https://github.com/excalidraw/excalidraw/pull/3230
https://github.com/twbs/bootstrap/pull/33334 (non default branch)

## Screenshot
image

### CODE REVIEW

File: source/features/clean-conversation-headers.tsx [hunk 1]
```diff
@@ -41,12 +41,8 @@ function initPR(): void {
 				byline.prepend('by ');
 			}
 
-			if (isDefaultBranch || (pageDetect.isClosedPR() && baseBranch.title.endsWith(':master'))) {
-				// Removes: octocat wants to merge 1 commit into [github:dev] from octocat:feature
-				baseBranch.hidden = true;
-			} else {
-				// Add back "into" if the PR base branch is not the default branch
-				baseBranch.before(' into ');
```
Comment (fregante) [inline source/features/clean-conversation-headers.tsx:49]: Comments still needed

**Comment by fregante** [comment]
It’s best to find the original PR and test on every situation in that PR, not just 2

**Comment by yakov116** [comment]
> It’s best to find the original PR and test on every situation in that PR, not just 2

Sorry I did 2 pr's at once and did not do this one correctly.

**Comment by yakov116** [comment]
Closing this, allowing anyone to take over.

**Pull request closed without merging**